### PR TITLE
fix(code-snippet): update to ibm-plex-mono

### PR DESCRIPTION
**Pull request by tw15egan**
Make sure `code-snippet` is using `ibm-plex-mono`

### CODE REVIEW

File: src/components/code-snippet/_mixins.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @mixin bx--snippet {
-  font-family: monospace;
+  font-family: 'ibm-plex-mono', monospace;
```
Comment (marijohannessen) [inline src/components/code-snippet/_mixins.scss]: Can you change the fallbacks to `'ibm-plex-mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace`?

Comment (marijohannessen) [reply src/components/code-snippet/_mixins.scss]: That's what is in the IBM Type repo

**Comment by marijohannessen** [comment]
👍 ✅ ❗️

File: src/components/code-snippet/_mixins.scss (hunk 1 above)
Comment (tw15egan) [reply src/components/code-snippet/_mixins.scss]: `DeJaVu SaNs MoNo`

**Comment by marijohannessen** [comment]
Since the change is only 1 line, you can merge this once the tests has passed.

**Pull request merged**